bin/wp-cli.php

```diff
@@ -128,8 +127,13 @@ public function flush() {
 	 *      # Index six specific posts
 	 *      wp searchpress index 12340 12341 12342 12343 12344 12345
 	 *
+	 *      # Index posts published between 11/1/2015 and 12/15/2015 inclusive
+	 *      wp searchpress index --from=11012015 --to=12152015
+	 *
+	 *      # Index posts published after 11/1/2015
+	 *      wp searchpress index --from=11012015
 	 *
-	 * @synopsis [--flush] [--put-mapping] [--bulk=<num>] [--limit=<num>] [--page=<num>] [<post-id>]
+	 * @synopsis [--flush] [--put-mapping] [--bulk=<num>] [--limit=<num>] [--page=<num>] [--from=<mmddyyyy>] [--to=<mmddyyyy>] [<post-id>]
```

Comment:
the standard format for passing dates around in wp-cli is `YYYY-MM-DD`. For the purposes of documentation, make this `[--from=<date>] [--to=<date>]` and in the documentation you're going to add, specify that it should be in `YYYY-MM-DD` format